MoltenVK/MoltenVK/GPUObjects/MVKBuffer.mm

```diff
@@ -212,6 +212,12 @@
     return _mtlBufferCache;
 }
 
+#if MVK_XCODE_14
+uint64_t MVKBuffer::getMTLBufferAddress() {
+	return [_mtlBuffer gpuAddress] + getMTLBufferOffset();
+}
+#endif
```

Comment:
It seems it would be cleaner if you move the `MVK_XCODE_14` test to inside this function, return `0` in the `#else` part...and then remove the `MVK_XCODE_14` from both the declaration in the `.h` file, and the `vkGetBufferDeviceAddressKHR()` implementation in `vulkan.mm`.

Basically...expose can call the function always, and let the function handle what to do.